api-reference/_hidden/dxDataGridColumnButton/disabled.md

```diff
@@ -5,22 +5,22 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
-
+Specifies whether the button is disabled.
+ 
 ##### param(options): Object
-<!-- Description goes here -->
+Information about the current row and column that contain the button.
```

Comment:
`row and column that contain the button` seems to imply both the column and thr row have a button